docs/index.md

```diff
@@ -45,232 +45,69 @@ theme:
-While not strictly required, it is *highly* recommended to install
-napari into a clean virtual environment using an environment manager like
-[conda](https://docs.conda.io/en/latest/miniconda.html) or
-[venv](https://docs.python.org/3/library/venv.html). 
-This should be set up before you install napari. For example, setting with
-up a Python {{ python_version }} environment with `conda`:
-
-{{ conda_create_env }}
-````
-
-### From conda
-
-Installation from conda-forge using conda or mamba is required for newer, arm64 macOS machines
-(Apple Silicon):
-
-```sh
-conda install -c conda-forge napari
-```
-
-(from-source)=
-### Current development branch from github
-
-To install the current `main` branch on github (which will usually be ahead
-of the latest release on pypi)
-
-```sh
-pip install "git+https://github.com/napari/napari.git#egg=napari[all]"
-```
-
-For more information or troubleshooting see our [installation
-tutorial](./tutorials/fundamentals/installation)
-
-````{admonition} Specifying a GUI Backend
-:class: tip
-
-napari needs a library called [Qt](https://www.qt.io/) to run its user interface
-(UI). In Python, there are two alternative libraries to run this, called
-[PyQt5](https://www.riverbankcomputing.com/software/pyqt/download5) and
-[PySide2](https://doc.qt.io/qtforpython/). By default, we don't choose for you,
-and simply running `pip install napari` will not install either. You *might*
-already have one of them installed in your environment, thanks to other
-scientific packages such as Spyder or matplotlib. If neither is available,
-running napari will result in an error message asking you to install one of
-them.
-
-Running `pip install "napari[all]"` will install the default framework--currently
-PyQt5, but this could change in the future.
-
-To install napari with a specific framework, you can use:
-
-```sh
-pip install "napari[pyqt5]"    # for PyQt5
-
-# OR
-pip install "napari[pyside2]"  # for PySide2
-```
-````
-
-## Getting started
-
- If you are new to napari, check out our
- [getting started guide](tutorials/fundamentals/getting_started.md).
-
-### Example
-
-(The example below requires the `scikit-image` package to run. We just use data
-samples from this package for demonstration purposes. If you change the example
-to use your own dataset, you may not need to install this package.)
-
-From inside an IPython shell or jupyter notebook you can open up an interactive
-viewer by calling:
-
-```python
-from skimage import data
-import napari
-
-viewer = napari.view_image(data.astronaut(), rgb=True)
-```
-
-![image](./images/screenshot-add-image.png)
-
-To do the same thing inside a script call (note the call to {func}`napari.run` at
-the end)
-
 ```python
-from skimage import data
+from skimage.data import cells3d
 import napari
 
-viewer = napari.view_image(data.astronaut(), rgb=True)
-napari.run()  # start the event loop and show viewer
+viewer, layers = napari.imshow(cells3d(), channel_axis=1)
 ```
```

Comment:
Why leave here the code sample without any comment?

Reply:
I agree, comments in the code could be nice, but also at least 1 line of lead in between `napari` and the code block would be good for me.
Also, this code doesn't make that image 🤣

Reply:
@Czaki I was inspired by matplotlib.org which shows a (much tinier) code snippet together with a plot type. 

<img width="1312" alt="Screen Shot 2023-02-02 at 7 10 46 pm" src="https://user-images.githubusercontent.com/492549/216267660-c9bc3593-e5b8-4eee-b477-b05b7bedcaa1.png">

However I acknowledge that the effect here is awkward because of the size of the code snippet! 😅 When I showed the matplotlib inspiration in the docs meeting @psobolewskiPhD said, "Ok but that is *way* cooler!" 😂 and he's not wrong 😂 

Perhaps I reduce it to:

> Just
> ```
> import napari
> viewer, layer = napari.imshow(cells, channel_axis=1)
> ```
> to get:
> [video]

Alternatively, I can try to fit a caption under the image (need to check how to do that in myst):

[video/image]
viewer, layer = napari.imshow(cells, channel_axis=1)

But, I like "import napari" as a thing that people see early. 😊 

Thoughts?

Reply:
the equivalent of matplotlib will be:

```python
napari.imshow(cells, channel_axis=1)
``` 

As we can see also matplotlib omits an important part of the code.
Also, on video, it will be nice to show something more than just an image, but also other types of data.